### ISSUE TITLE
Feature: Set YAML Merge Keys via yaml-set

### ISSUE THREAD

**Issue by wwkimball** [wwkimball/yamlpath#113]
**Is your feature request related to a problem? Please describe.**
[YAML Merge Keys](https://yaml.org/type/merge.html) are extremely useful for eliminating structure and data duplication within YAML files.  At present, these are understood and preserved during all get/set/merge operations.  However, there is presently no means within yamlpath's command-line tools of creating a new YAML Merge Key within a file which does not already have one.

**Describe the solution you'd like**
A command like the following should be possible:  `yaml-set --change=/some/hierarchy/targets/* --mergekey=/merge_source --anchor=new_anchor_name`

This could change:

```yaml
---
merge_source:
  hash: with
  some: default
  reusable: values
some:
  hierarchy:
    targets:
      one:
        novel: structure
      two:
        novel: data
      three:
        overrides: defaults
        some: non-default
```

into:

```yaml
merge_source: &new_anchor_name
  hash: with
  some: default
  reusable: values
some:
  hierarchy:
    targets:
      one:
        <<: *new_anchor_name
        novel: structure
      two:
        <<: *new_anchor_name
        novel: data
      three:
        <<: *new_anchor_name
        overrides: defaults
        some: non-default
```

**Describe alternatives you've considered**
I have considered [another request which asks for the ability to share such default configuration by way of applying novel key-value pairs to change targets only when the specified key does not already exist](#107).  While this alternative would solve the topical issue, it does not employ the far more useful YAML Merge Keys, which can be used to set more than one default and facilitate changing defaults later as needed via a single data-point.  This helps preserve the discipline of, "One version of the truth."

**Comment by wwkimball** [comment]
I'm moving this to 3.6.0 because I've lost a couple of days for 3.5.0 due to Travis-CI turning its back on FOSS.  This forced me to spend considerable time migrating to GitHub Actions.

**Comment by wwkimball** [comment]
I have published [version 3.6.0](https://github.com/wwkimball/yamlpath/discussions/136) which includes this new capability, and more.
